templates/land-see/kim-myeong.amp.html

```diff
@@ -55,15 +55,147 @@
-      </p>
-      <p class="py3">
-        Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit, sed quia non numquam eius modi tempora incidunt ut labore et dolore magnam aliquam quaerat voluptatem. Ut enim ad minima veniam, quis nostrum exercitationem ullam corporis suscipit laboriosam, nisi ut aliquid ex ea commodi consequatur? Quis autem vel eum iure reprehenderit qui in ea voluptate velit esse quam nihil molestiae consequatur, vel illum qui dolorem eum fugiat quo voluptas nulla pariatur?
+    {{#land-see-story-content}}
+    <section class="land-see-story-detail relative mx-auto mb4 pt2">
+      {{#land-see-post-detail}}
+      <div class="mx-auto center {{post-classes}}">
+        <!-- {{#image}}
+        <amp-img
+          class="{{classes}}"
+          alt="{{alt-text}}"
+          width="{{width}}"
+          height="{{height}}"
+          layout="{{layout}}"
+          src="{{src}}">
+        </amp-img>
+        {{/image}} -->
+        <a class="land-see-post-category underline caps max-width-1 pb1" href="{{post-category-url}}">{{post-category}}</a>
+        <h1 class="land-see-hero-title pt2 mx2"><a class="text-decoration-none" href="{{post-url}}">{{{post-title}}}</a></h1>
+        <p class="land-see-hero-caption mx-auto max-width-2 pt1 px2 mb3">
+          {{{post-copy}}}
+        </p>
+      </div>
+      {{/land-see-post-detail}}
+
+      {{#land-see-copy}}
+        {{#subheader}}
+        <h2 class="land-see-story-subheader max-width-3 mx-auto pb0 pt4 px4">{{.}}</h2>
+        {{/subheader}}
+        {{#paragraph}}
+        <p class="land-see-story-copy max-width-3 mx-auto py2 px4">
+          {{{.}}}
+        </p>
+        {{/paragraph}}
+        {{#blockquote}}
+        <blockquote class="land-see-blockquote max-width-3 mx-auto py4 px4 {{classes}} "{{#cite-url}}cite="{{.}}"{{/cite-url}}>
+          <p>{{quote}}</p>
+          {{#citation}}
+          <cite>{{.}}</cite>
+          {{/citation}}
+        </blockquote>
+        {{/blockquote}}
+      {{/land-see-copy}}
+
+      {{#land-see-image-gallery}}
+      <div class="land-see-image-gallery mx-auto">
+        <div class="clearfix mx1">
```

Comment:
I dont think you need this wrapper div

Reply:
I needed it to apply margin to the horizontally scrollable preview/progress image gallery navigation. Since I wanted it to apply to the gallery itself, as well, I applied the div to the carousel _and_ navigation markup. I'll bump the carousel out and give it its own `mx1` class, but keep the wrapper div on the `land-see-gallery-progress` to preserve the desired margin behavior:

<img width="345" alt="screen shot 2017-10-24 at 6 18 24 pm" src="https://user-images.githubusercontent.com/523157/31972961-fa69c582-b8e8-11e7-9298-ddc4bd7765d7.png">